pkg/services/sqlstore/migrator/dialect.go

```diff
@@ -45,6 +45,7 @@ type Dialect interface {
 	PostInsertId(table string, sess *xorm.Session) error
 
 	CleanDB() error
+	TruncateDB() error
```

Comment:
None of the existing interface methods have comments, but maybe this would be a good time to start? In particular, what is the difference between CleanDB and TruncateDB seems like a worthy comment. 

Maybe TruncateDBTables is more specific?